buildlib/pr/tests.yml

```diff
@@ -23,11 +23,12 @@ jobs:
     ${{ if parameters.container }}:
       container: ${{ parameters.container }}
     workspace:
-      clean: all
+      clean: outputs
     steps:
       - checkout: self
         clean: true
         fetchDepth: 100
+        retryCountOnTaskFailure: 5
```

Comment:
there are other places we check out code, shall we add retryCountOnTaskFailure: 5 in all those places as well?

Reply:
Added retry for all checkout steps.